QEfficient/exporter/export_hf_to_cloud_ai_100.py

```diff
@@ -221,18 +209,15 @@ def convert_to_cloud_kvstyle(
 
 
 def export_kvstyle_transformed_model_to_onnx(model_name: str, transformed_model: torch.nn.Module, tokenizer: Union[PreTrainedTokenizer, PreTrainedTokenizerFast],
-                                          onnx_dir_path: str, seq_len: int, save_fp32_onnx: Optional[bool] = False, save_fp16_onnx: Optional[bool] = True):
-    
-    if tokenizer.padding_side != "left":
-        logger.warning("Please use padding_side='left' while initializing the tokenizer")
-        tokenizer.padding_side = "left"
-
-    tokenizer.pad_token_id = tokenizer.eos_token_id if tokenizer.pad_token_id is None else tokenizer.pad_token_id    
+                                          onnx_dir_path: str, seq_len: int, save_fp32_onnx: Optional[bool] = False, save_fp16_onnx: Optional[bool] = True):  
```

Comment:
why save fp16 and not fp32?

Reply:
That's the default behavior to save `fp16_clipped onnx` file, as this is the file used for `qpc` compilation. -> saves storage space.
Whereas we let users save `fp32 onnx` file as well, if he wants to do so via low level APIs

Reply:
Agree

Reply:
Initially, the default case was that we were saving both the clipped and unclipped versions of the ONNX model, which caused space issues for larger models. However, at the time of compilation, we were using the clipped version of the ONNX model. To handle this situation, we have enabled this feature, where save_fp32 will delete the unclipped version, as its default value is False. It gives user the option to save both, only clipped and only unclipped versions of the ONNX model.